plugins/metadata/config/schema-transformed.xml

```diff
@@ -1,2 +1,2 @@
oIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="metadata_profile_id" type="integer"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="metadata_profile_version" type="integer"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="partner_id" type="integer"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="label" type="varchar" size="127"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="key" type="varchar" size="127"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="type" type="varchar" size="127"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="xpath" type="varchar" size="255"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="status" type="tinyint"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="related_metadata_profile_id" type="integer"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="custom_data" type="longvarchar"/><index name="partner_id"><index-column name="partner_id"/></index><index name="profile_id_and_version"><index-column name="metadata_profile_id"/><index-column name="metadata_profile_version"/></index></table><table skipSql="false" abstract="false" name="metadata"><column lazyLoad="false" name="id" type="integer" required="true" primaryKey="true" autoIncrement="true"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="created_at" type="timestamp"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="updated_at" type="timestamp"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="version" type="integer"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="metadata_profile_id" type="integer"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="metadata_profile_version" type="integer"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="partner_id" type="integer"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="object_id" type="varchar" size="20"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="object_type" type="integer"/><column autoIncrement="false" primaryKey="false" required="false" lazyLoad="false" name="status" type="tinyint"/><index name="partner_id"><index-column name="partner_id"/></index><index name="profile_id_and_version"><index-column name="metadata_profile_id"/><index-column name="metadata_profile_version"/></index><index name="object_id_and_type"><index-column name="object_type"/><index-column name="object_id"/></index></table></database>
```

Comment:
@noam-arad why do you have 2 opening tags for XML?
First in line 1 and the new one you added at line 2.
I think that one from line 2 should be removed "<?xml version="1.0"?>"